nifi-nar-bundles/nifi-solr-bundle/nifi-solr-processors/src/test/resources/solr/testCollection/conf/schema.xml

```diff
@@ -16,6 +16,16 @@
     <field name="marks" type="int" indexed="true" stored="true" />
     <field name="test" type="string" indexed="true" stored="true" />
     <field name="subject" type="string" indexed="true" stored="true" />
+
     <field name="created" type="date" indexed="true" stored="true" />
 
+    <field name="id" type="string" indexed="true" stored="true" />
+    <field name="double_single" type="double" indexed="true" stored="true" />
+    <field name="integer_single" type="int" indexed="true" stored="true" />
+    <field name="integer_multi" type="int" indexed="true" stored="true"  multiValued="true"/>
+    <field name="string_single" type="string" indexed="true" stored="true" />
+    <field name="string_multi" type="string" indexed="true" stored="true" multiValued="true"/>
+
+    <uniqueKey>id</uniqueKey>
```

Comment:
What if Solr doc doesn't have an uniqueKey? Does this processor still work without uniqueKey??

Reply:
the uniqueKey field has to be part of the sorting. Well-configured Solr indexes always include this kind of field as many things will not work properly without this field. Actually, I have never seen a Solr index without this (and I have seen a lot ... ;).

Reply:
I agree with that, most indices have unique key. But just asked it because it is not mandatory to have an unique key according to [Solr documentation](https://wiki.apache.org/solr/SchemaXml#The_Unique_Key_Field). Then I prefer to state that unique key is required for this processor to work properly in NiFi documentation.